views/Todo/Todo.js

```diff
@@ -1,17 +1,63 @@
 import React from "react";
-import { View } from "react-native";
+import { Text, View } from "react-native";
 import Error from "../../components/Error/Error";
 import Header from "../../components/Header/Header";
 import Loading from "../../components/Loading/Loading";
 import TextArea from "../../components/TextArea/TextArea";
 
 export default function Todo({ loading, error, data, insets }) {
-  if (loading) return <Loading />;
-  if (error) return <Error message={error.message} />;
+  if (loading)
+    return (
+      <View
```

Comment:
you use this same wrapper view 4 times. Lets try and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up a bit but having a singe wrapper and using 
`{loading && <Loading/>}`
`{error && <Error/>}`

etc.